src/app/core/services/personal-cards.service.ts

```
@@ -350,7 +350,22 @@ export class PersonalCardsService {
+    const payload = {
+      data: {
+        personal_card_id: accountId,
+      },
+    };
+    return this.spenderPlatformV1ApiService.post('/personal_card_transactions', payload);
+  }
+
+  fetchTransactions(
+    accountId: string,
+    usePlatformApi: boolean
+  ): Observable<ApiV2Response<PersonalCardTxn> | PlatformApiResponse<{}>> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)